glfs.c

```diff
@@ -430,58 +430,6 @@ static char* tcmu_get_path( struct tcmu_device *dev)
 	return config;
 }
 
-
-static bool glfs_check_config(const char *cfgstring, char **reason)
-{
```

Comment:
For now we have these two stages to create a target using targetcli:
1. Create the backend with glfs storage module
    targetcli /backstores/user:glfs create glfsLUN 40G block@10.70.42.151/elastic-media.img
2. Set/Export LUN
    targetcli /iscsi/$IQN/tpg1/luns create /backstores/user:glfs/glfsLUN
    
glfs_check_config() checks, if the target file exists or not at stage 1, as anyway we are doing that as part of tcmu_glfs_open() and ensuring the size as-well, in case if file doesn't exist we fail in stage 2, and we don't succeed target creation.

In reality glfs_check_config() is expensive as it involves network operations :-)